pkg/project/registry/project/proxy/proxy.go

```diff
@@ -167,6 +182,34 @@ func (s *REST) Update(ctx kapi.Context, name string, objInfo rest.UpdatedObjectI
 	return projectutil.ConvertNamespace(namespace), false, nil
 }
 
+func (s *REST) tryImpersonatingNamespaceUpdate(project *api.Project, ctx kapi.Context) (*kapi.Namespace, bool) {
+	user, ok := kapi.UserFrom(ctx)
+	if !ok {
+		return nil, false
+	}
+
+	impersonatingClient, err := authclient.NewImpersonatingKubernetesClientset(user, s.config)
```

Comment:
I guess both you and @pweil- voiced that concern in the main thread. I'm looking from an audit point of view ;)